Bindings/Python/tests/test_sockets_inputs_outputs.py

```diff
@@ -228,22 +228,53 @@ def test_connecting_and_iterate_inputs(self):
                 rep.getInput('inputs')).getValue(s, 3) == 2.0)
 
 
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
+    def test_input_alias(self):
+        model_filename = 'wiring_inputs_and_outputs_with_TableReporter.osim'
```

Comment:
Could you rename this so it doesn't get confused with the python example of this name? Perhaps `test_input_alias.osim`.